api/python/quilt3/packages.py

```diff
@@ -1668,6 +1668,10 @@ def verify(self, src, extra_files_ok=False):
         if src_dict and not extra_files_ok:
             return False
 
+        for lk, e in self.walk():
+            if e.hash["type"] != "SHA256":
+                raise QuiltException(f"Unsupported hash type: {e.hash['type']!r}. Try to update quilt3.")
```

Comment:
Rather than using a string for the hash type (`"SHA256"`), define it as a variable, then output the required hash type in the exception message so it's more helpful to the user.